README.md

```diff
@@ -464,12 +464,10 @@ make delete
    To hit the indexer (not the service) with multiple notification requests, run
 
    ```
-   python scripts/reindex.py --workers=1 --sync --indexer-url http://127.0.0.1:8000/
+   python scripts/reindex.py --workers=1 --indexer-url http://127.0.0.1:8000/
    ```
 
-   in a separate shell. The `--sync` argument causes the Chalice app to invoke
-   the indexing code directly instead of queuing an SQS message to be consumed
-   by the indexer worker Lambda function in AWS.
+   in a separate shell.
```

Comment:
We need to remove everything starting with "To hit the indexer" because this does require a complete deployment lambda, queues, ES and all. And worse, it would only do the queuing into the notifications queue locally, the rest of indexing (creating contributions, writing aggregates) would happen in the lambdas. Interestingly, this would also apply to `--sync` where the contributions would be created locally and the aggregate would be created by the lambda. All in all, a mess. So it's a good thing we remove --sync. To test the indexer locally, you just need a write a test. We should also rename the section to `Running service locally`.